tests/scripts/create-dev-cluster.sh

```diff
@@ -0,0 +1,127 @@
+#!/bin/bash
+
+#set -eEx
+KUBECTL="minikube kubectl --"
+ROOK_EXAMPLES_DIR="../../deploy/examples/"
```

Comment:
maybe a step on how and where to run this, with the doc update.

Reply:
Added some docs on where to find the script. About where they can run the script I think it's self explanatory as the script by default will pick the example dir, otherwise the user has to provide the path and the script will check if the provided path is correct or not. I also added a usage/help function.